grammar.js

```diff
@@ -751,6 +755,14 @@ module.exports = grammar({
       $._automatic_semicolon
     ),
 
+    null_literal: $ => token('null'),
```

Comment:
I don't think you need the `token()` call around the string literal.

That function is only needed when you're using other rule-builder functions like `seq`/`repeat`/`choice`, and you want to tell Tree-sitter that the whole rule should be a single token. With a string literal, it's already a single token by default.